lib/signs_ui/config/state.ex

```diff
@@ -47,6 +56,11 @@ defmodule SignsUi.Config.State do
         save_sign_config_changes(%{}, config)
         save_configured_headways_changes(config.configured_headways, config)
 
+        save_chelsea_bridge_announcements(
+          Map.get(config, :chelsea_boarding_announcements, "off"),
```

Comment:
Is this supposed to be "boarding"? I can't seem to find the corresponding test, if there is one.

Reply:
yeah, should be

Reply:
Hm, why didn't any tests fail when this value was incorrect? I guess we must have test coverage of this line in some way or else Codecov would have complained, but maybe we're not checking the actual value of the saved config.

Reply:
the logic was tested [here](https://github.com/mbta/signs_ui/blob/fl/chelsea/test/signs_ui/config/state_test.exs#L162-L163), and happened to pass because of the fallback to `"off"` in the `Map.get` statement above. I flipped the order of the assertions, which caused the test to fail before I changed the key.